### DISCUSSION TITLE
How to custom property type radio StyleManager

### DISCUSSION THREAD

**Discussion by alemenciones** [GrapesJS/grapesjs#4044] [Q&A]
I want add to the styleManager a radio group like "text-align", but to change margin-left|right properties

radio button "left": margin-left: 0 !important;
radio button "center": margin-left: auto !important; and margin-right: auto !important; 
radio button "right": margin-left: auto !important; and margin-right: 0 !important; 

maybe the way is setting just margin, but I need to keep the values that the user has set for top and bottom (n)

radio button "left": margin: n 0 n n !important;
radio button "center": margin: n auto n auto !important; 
radio button "right": margin: n auto n 0 !important;

**Comment by aliibrahim123** [comment]
you can use `margin-left` property without `margin`

**Reply by alemenciones** [reply]
if i want align to right, i need to use margin-right too:

margin-left: auto; and margin-right: 0;

**Reply by senthil177cse** [reply]
Hi @alemenciones ...
    Did you created Radio button group component? if yes can you explain that please...
